formats/developers/compression.txt

```diff
@@ -0,0 +1,170 @@
+| OME-TIFF, zipped          | Great – 122,031 bytes      | Poor – The appropriate OME-TIFF file must be uncompressed from the    |
+|                           |                            | archive.                                                              |
++---------------------------+----------------------------+-----------------------------------------------------------------------+
+| OME-TIFF+LZW, zipped      | Good – 134,834 bytes       | Poor – The appropriate OME-TIFF file must be uncompressed from the    |
+|                           |                            | archive, and the plane must be decoded from LZW.                      |
++---------------------------+----------------------------+-----------------------------------------------------------------------+
+| OME-TIFF+LZW, 7-zipped    | Good – 135,014 bytes       | Poor – The appropriate OME-TIFF file must be uncompressed from the    |
+|                           |                            | archive, and the plane must be decoded from LZW.                      |
++---------------------------+----------------------------+-----------------------------------------------------------------------+
+| OME-XML, gzipped          | Great – 124,763 bytes      | Worst – Entire dataset must be uncompressed, then the plane must be   |
+|                           |                            | decoded from base64.                                                  |
++---------------------------+----------------------------+-----------------------------------------------------------------------+
+| OME-XML, zipped           | Great – 124,764 bytes      | Worst – Entire dataset must be uncompressed, then the plane must be   |
+|                           |                            | decoded from base64.                                                  |
++---------------------------+----------------------------+-----------------------------------------------------------------------+
+| OME-XML, 7-zipped         | Great – 125,830 bytes      | Worst – Entire dataset must be uncompressed, then the plane must be   |
+|                           |                            | decoded from base64.                                                  |
++---------------------------+----------------------------+-----------------------------------------------------------------------+
+
+The performance penalty for accessing individual image planes from
+externally compressed formats (zip, gzip, and 7-zip) is high, since the
+data must be decompressed. There is little penalty for accessing them
+from uncompressed OME-XML or OME-TIFF—with OME-XML, file readers can
+build a list of offsets by skipping past the bulk of the BinData
+characters according to the Length attribute values, and with OME-TIFF,
+file readers can seek to the offsets indicated in the IFD entries.
+
+Accessing them from an uncompressed OME-TIFF file, however, is
```

Comment:
What is "them".  Since it's a separate paragraph, it might be clearer to be explicit.